static/elements/chromedash-timeline.js

```diff
@@ -234,42 +237,44 @@ class ChromedashTimeline extends LitElement {
   }
 
   _renderHTTPArchiveData() {
-    if (!this.props.length) {
-      return;
-    }
+    if (!this.props.length) return;
 
     const feature = this.props.find((el) => el[0] === parseInt(this.selectedBucketId));
     if (feature) {
       let featureName = feature[1];
-      const inputEl = this.shadowRoot.getElementById('datalistinput');
+      const inputEl = this.shadowRoot.querySelector('#datalist-input');
       inputEl.value = featureName;
 
       if (this.type == 'css') {
         featureName = convertToCamelCaseFeatureName(featureName);
       }
       const REPORT_ID = '1M8kXOqPkwYNKjJhtag_nvDNJCpvmw_ri';
       const dsEmbedUrl = `https://datastudio.google.com/embed/reporting/${REPORT_ID}/page/tc5b?params=%7B"df3":"include%25EE%2580%25800%25EE%2580%2580IN%25EE%2580%2580${featureName}"%7D`;
-      const hadEl = this.shadowRoot.getElementById('httparchivedata');
+      const hadEl = this.shadowRoot.querySelector('#http-archive-data');
       hadEl.src = dsEmbedUrl;
 
-      const bigqueryEl = this.shadowRoot.getElementById('bigquery');
+      const bigqueryEl = this.shadowRoot.querySelector('#bigquery');
       bigqueryEl.textContent = `#standardSQL
-SELECT yyyymmdd, client, pct_urls, sample_urls
-FROM \`httparchive.blink_features.usage\`
-WHERE feature = '${featureName}'
-ORDER BY yyyymmdd DESC, client`;
+        SELECT yyyymmdd, client, pct_urls, sample_urls
```

Comment:
Does this add visible space that would be copied onto the clipboard if the user tries to use this code?

Reply:
It actually does, so I have changed it back.